docs/dev/publicdashboard/Working_with_public_dashboard.md

```diff
@@ -0,0 +1,159 @@
+
+What if we want to load a different dataset than staging, and test for a different program/study.
+Let us consider the example below:
+```
+services:
+    notebook-server:
+        environment:
+        - DB_HOST=mongodb://db/openpath_prod_usaid_laos_ev
+        - WEB_SERVER_HOST=0.0.0.0
+        - CRON_MODE=TRUE
+        - STUDY_CONFIG=usaid-laos-ev
+```
+1. For testing different config: Change the STUDY_CONFIG, as
+ `STUDY_CONFIG=<program/study type>`
+ Example: `STUDY_CONFIG=stage-program`
+1. Similarly, change the `DB_HOST=<dataset>`
+Example: `DB_HOST=mongodb://db/openpath_prod_usaid_laos_ev`
+to load the dataset `openpath_prod_usaid_laos_ev` which has been loaded into the MongoDB.
+
+# Execution of different public dashboard notebooks automatically (without launching Jupyter notebook)
+---
+
+``` 
+$ docker-compose -f docker-compose.yml build
+$ docker-compose -f docker-compose.yml  up 
+$ docker ps
+$ docker exec -it em-public-dashboard-notebook-server-1 /bin/bash
+$ root@06f07def2c0e:/usr/src/app# source setup/activate.sh
+$ (emission) root@06f07def2c0e:/usr/src/app# cd saved-notebooks
+```
+
+Execution of notebooks
+```
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/update_mappings.py mapping_dictionaries.ipynb
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py generic_metrics.ipynb default
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py generic_metrics_sensed.ipynb default
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py generic_timeseries.ipynb default
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py mode_specific_metrics.ipynb default
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py mode_specific_timeseries.ipynb default
+$ (emission) root@06f07def2c0e:/usr/src/app/saved-notebooks# PYTHONPATH=.. python bin/generate_plots.py energy_calculations.ipynb default
+```
+The above script will generate all the charts from all the notebooks. Execute one notebook script at a time.
+
+Then go to http://localhost:3274/
+
+- If you want to test for specific configuration (program/study):
+For example, if you want to test for usaid-laos-ev:
+    http://localhost:3274/?study_config=usaid-laos-ev
+and test.
+
+Note: Make sure the Year and Month parameter for the executing notebook is not None. If it's set to None, it will not generate the charts.
+
+# Launch and explore the database
+```
+$ docker-compose -f docker-compose.yml  up 
+$ docker ps
+$ docker exec -it em-public-dashboard-db-1 mongo
+$ show dbs
+$ use <database_name>
+$ show collections
+```
+
+Most useful collection: `Stage_analysis_timeseries`
+
+Better way to explore and modify database:
+- Through the Server code
+
```

Comment:
If this is the better way to explore the database, why is the other one listed, and why is this second? Please remove the instructions to use mongodb directly

Reply:
Removed instructions to use mongodb directly.